src/Zone.js

```diff
@@ -35,17 +34,31 @@ export default class Zone extends UXBase {
 		return mark;
 	});
 
+	// Parent View object this zone belongs to
```

Comment:
Nitpick, but let's move this to the top of the class after where `measureName` is defined.

Reply:
Removed now.